FBSnapshotTestCase/Categories/UIApplication+StrictKeyWindow.m

```diff
@@ -15,12 +15,8 @@ @implementation UIApplication (StrictKeyWindow)
 - (UIWindow *)fb_strictKeyWindow
 {
   UIWindow *keyWindow = [UIApplication sharedApplication].keyWindow;
-  if (!keyWindow) {
-    [NSException raise:@"FBSnapshotTestCaseNilKeyWindowException"
-                format:@"Snapshot tests must be hosted by an application with a key window. Please ensure your test"
-                        " host sets up a key window at launch (either via storyboards or programmatically) and doesn't"
-                        " do anything to remove it while snapshot tests are running."];
-  }
+  NSString *message = @"Snapshot tests should be hosted by an application with a key window. Please ensure your test host sets up a key window at launch (either via storyboards or programmatically) and doesn't  do anything to remove it while snapshot tests are running.";
```

Comment:
Why to log this message if `keyWindow` exists? Maybe you should keep it in `if` statement?
Also, you didn't update a comment in a header file about raising an exception.
(not sure if this method should still contain `strict` word in the name...)